plio/io/io_controlnetwork.py

```diff
@@ -29,6 +32,71 @@ def write_filelist(lst, path="fromlist.lis"):
         handle.write('\n')
     return
 
+
+class MessageType(IntEnum):
```

Comment:
These are only associated with Measures, I expect that point logging will get added somewhere down the line. Rename to 'MeasureMessageType'?